query/src/main/java/org/infinispan/configuration/cache/PrivateIndexingConfigurationBuilder.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.configuration.cache;
+
+import org.infinispan.commons.configuration.Builder;
+import org.infinispan.commons.configuration.Combine;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+
+public class PrivateIndexingConfigurationBuilder extends AbstractModuleConfigurationBuilder implements Builder<PrivateIndexingConfiguration> {
```

Comment:
You should add the /** @api.private **/ doclet to avoid inclusion in javadoc

Reply:
good idea... I'm applying it